<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageMember</class>
 <widget class="QDialog" name="ManageMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Manage Member</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_first">
            <property name="text">
             <string>First Name:</string>
            </property>
            <property name="buddy">
             <cstring>line_phone</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_first"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_last">
            <property name="text">
             <string>Last Name:</string>
            </property>
            <property name="buddy">
             <cstring>line_phone</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_last"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_email">
            <property name="text">
             <string>Email Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_email"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_password">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_password"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_author">
            <property name="text">
             <string>Phone Number:</string>
            </property>
            <property name="buddy">
             <cstring>line_phone</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_phone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_home_1">
            <property name="text">
             <string>Home Address 1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_home_1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_home_2">
            <property name="text">
             <string>Home Address 2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_home_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_code">
            <property name="text">
             <string>Post Code:</string>
            </property>
            <property name="buddy">
             <cstring>line_phone</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_code"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
